go/glide.lock

```diff
@@ -174,7 +174,6 @@ imports:
 - name: github.com/satori/go.uuid
   version: b061729afc07e77a8aa4fad0a2fd840958f1942a
 - name: github.com/Sirupsen/logrus
-  version: 4b6ea7319e214d98c938f12692336f7ca9348d6b
```

Comment:
I think you should run `make update-vendor` to regenerate this file with the new commit hash.  I'm not sure it's valid as it stands.